Makefile

```diff
@@ -48,52 +51,133 @@ cli:
 	cd sdk-java && docker build -f Dockerfile.quarkus -t $(KAR_JAVA_REACTIVE_RUNTIME) --build-arg KAR_BASE=$(KAR_BASE) .
 
-dockerBuildExamples:
+dockerBuildCore:
+	make docker-python-sdk
+	make docker-js-sdk
+	make docker-java-sdk
+
+
+# BUILD EXAMPLE images
+docker-python-examples: docker-python-sdk
+	cd examples/actors-python && docker build -f Dockerfile.containerized --build-arg PYTHON_RUNTIME=$(KAR_PYTHON_SDK) -t $(KAR_EXAMPLE_ACTORS_PYTHON_CONTAINERIZED) .
+	cd examples/actors-python && docker build -f Dockerfile.cluster --build-arg PYTHON_RUNTIME=$(KAR_PYTHON_SDK) -t $(KAR_EXAMPLE_ACTORS_PYTHON_CLUSTER) .
+
+docker-js-examples: docker-js-sdk
 	cd examples/actors-dp-js && docker build --build-arg JS_RUNTIME=$(KAR_JS_SDK) -t $(KAR_EXAMPLE_JS_DP) .
 	cd examples/actors-events && docker build --build-arg JS_RUNTIME=$(KAR_JS_SDK) -t $(KAR_EXAMPLE_JS_EVENTS) .
 	cd examples/actors-ykt && docker build --build-arg JS_RUNTIME=$(KAR_JS_SDK) -t $(KAR_EXAMPLE_JS_YKT) .
 	cd examples/service-hello-js && docker build --build-arg JS_RUNTIME=$(KAR_JS_SDK) -t $(KAR_EXAMPLE_JS_HELLO) .
-	cd examples/unit-tests && docker build --build-arg JS_RUNTIME=$(KAR_JS_SDK) -t $(KAR_EXAMPLE_JS_TESTS) . 
+	cd examples/unit-tests && docker build --build-arg JS_RUNTIME=$(KAR_JS_SDK) -t $(KAR_EXAMPLE_JS_TESTS) .
+
+docker-java-examples: docker-java-sdk
 	cd examples/actors-dp-java && docker build --build-arg JAVA_BUILDER=$(KAR_JAVA_SDK) --build-arg JAVA_RUNTIME=$(KAR_JAVA_RUNTIME) -t $(KAR_EXAMPLE_JAVA_DP) .
 	cd examples/actors-dp-java-reactive && docker build --build-arg JAVA_BUILDER=$(KAR_JAVA_SDK) --build-arg JAVA_RUNTIME=$(KAR_JAVA_REACTIVE_RUNTIME) -t $(KAR_EXAMPLE_JAVA_REACTIVE_DP) .
 	cd examples/service-hello-java/server && docker build --build-arg JAVA_BUILDER=$(KAR_JAVA_SDK) --build-arg JAVA_RUNTIME=$(KAR_JAVA_RUNTIME) -t $(KAR_EXAMPLE_JAVA_HELLO) .
 
-dockerBuildBenchmarks:
+dockerBuildExamples:
+	make docker-js-examples
+	make docker-java-examples
+	make docker-python-examples
+
+dockerBuildBenchmarks: docker-js-sdk
 	cd benchmark/kar-bench && docker build --build-arg JS_RUNTIME=$(KAR_JS_SDK) -t $(KAR_BENCH_JS_IMAGE) .
 	cd benchmark/kafka-bench && docker build -t $(KAFKA_BENCH) .
 	cd benchmark/http-bench && docker build --build-arg JS_RUNTIME=$(KAR_JS_SDK) -t $(KAR_HTTP_BENCH_JS_IMAGE) .
 
-dockerPushCore:
+
+# PUSH CORE images
+docker-push-base:
 	docker push $(KAR_BASE)
 	docker push $(KAR_INJECTOR)
+
+docker-push-python-sdk: docker-push-base
+	docker push $(KAR_PYTHON_SDK)
+
+docker-push-js-sdk: docker-push-base
 	docker push $(KAR_JS_SDK)
+
+docker-push-java-sdk: docker-push-base
 	docker push $(KAR_JAVA_SDK)
 	docker push $(KAR_JAVA_RUNTIME)
 	docker push $(KAR_JAVA_REACTIVE_RUNTIME)
 
-dockerPushExamples:
+dockerPushCore:
+	make docker-push-python-sdk
+	make docker-push-js-sdk
+	make docker-push-java-sdk
+
+
+# PUSH EXAMPLES images
+docker-push-python-examples: docker-python-examples
+	docker push $(KAR_EXAMPLE_ACTORS_PYTHON_CONTAINERIZED)
```

Comment:
Not sure if it really matters, but does it make sense to publish the KAR_EXAMPLE_ACTORS_PYTHON_CONTAINERIZED image to quay.io?  Looking at what the image does, it seems useful for local testing and ci testing, but maybe not worth publishing to the quay.io namespace for kar.  If we don't see anyone every wanting to pull this image and run it (as opposed to building it locally), then maybe drop it from the push-python-examples target

Reply:
You're right, this is more useful for testing and just having it there as an example. Can definitely take it out.